packs/data/pathfinder-bestiary-3.db/azer.json

```diff
@@ -732,7 +729,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>The azer's Strikes deal an extra 1d6 fire damage (included in strikes).</p>\n<p>When the azer successfully performs a @Compendium[pf2e.action-macros.Grapple: Athletics]{Grapple: Athletics} or @Compendium[pf2e.action-macros.Shove: Athletics]{Shove: Athletics} action, they also deal [[/r {1d6}[fire]]]{1d6 fire damage} to their target.</p>"
+                    "value": "<p>The azer's Strikes deal an extra [[/r {1d6}[fire]]]{1d6 fire damage} (included in strikes).</p>\n<p>When the azer successfully performs a @Compendium[pf2e.actionspf2e.Grapple]{Grapple} or @Compendium[pf2e.actionspf2e.Shove]{Shove} action, they also deal [[/r {1d6}[fire]]]{1d6 fire damage} to their target.</p>"
```

Comment:
Why make the first formula rollable when it is already factored into the attacks?

Reply:
Good point - changed that